index.d.ts

```diff
@@ -15,6 +15,7 @@ interface FabProps {
   icon: ReactElement;
   event?: 'hover' | 'click';
   children?: ReactElement[] | ReactElement;
+  showTitle: boolean;
```

Comment:
A couple of things here:
1. The name `showTitle` implies that this must be set to `true` in order for the title to show, but what we really mean is "show the title without having to hover". Maybe a better name would be `alwaysShowTitle` or `persistTitle`?
2. This needs to be an optional prop `showTitle? : boolean`. We don't want to force the definition of a prop when we don't have to.

Reply:
- `alwaysShowTitle` is fine
- I'll remove the `showTitle?` prop definition